src/lib/merkle_ledger/test/test_mask.ml

```diff
@@ -741,6 +741,14 @@ module Make_maskable_and_mask_with_depth (Depth : Depth_S) = struct
     module Location = Location
     module Location_binable = Location_binable
     module Kvdb = In_memory_kvdb
+
+    type maps_t =
+      { accounts : Account.t Location.Map.t
```

Comment:
This type is repeated at least 6 times in this PR

Need to find a way to reduce number of repetitions.
Suggestion: define a module parametrized by `Location` that would define this type and include this module elsewhere instead of copying the type in plain.

Reply:
This is probably not doable as we can't have versioned type in functors.